server/src/main/java/com/project/yozmcafe/service/MemberService.java

```diff
@@ -22,4 +28,14 @@ public MemberResponse findById(final String id) {
 
         return MemberResponse.from(member);
     }
+
+    public List<LikedCafeResponse> findLikedCafesById(final String memberId, Pageable pageable) {
+        final Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new IllegalArgumentException("해당하는 회원이 존재하지 않습니다."));
+
+        final Slice<LikedCafe> likedCafes = memberRepository.findLikedCafeByMemberId(member.getId(), pageable);
+
+        return likedCafes.stream().map(LikedCafe::getCafe)
+                .map(cafe -> new LikedCafeResponse(cafe.getId(), cafe.getRepresentativeImage())).toList();
+    }
```

Comment:
```suggestion
return likedCafes.stream()
    .map(LikedCafeResponse::from)
    .toList();
```
위와 같은 형태면 좋을 것 같습니다!

Reply:
좋아용